webapp/components/wpt-bsf.js

```diff
@@ -152,9 +151,17 @@ class WPTBSF extends LoadingState(PolymerElement) {
     };
     this.enterChart = () => {
       this.isInteracting = true;
+      const event = new CustomEvent('is-interacting-changed', {
```

Comment:
we don't use "-" for the event name.  How about showbsf?
```suggestion
      const event = new CustomEvent('showbsf, {
```

Reply:
I used the same event name that Polymer fires when the isInteractive property changes, but I can use one without dashes (interactingchanged) if that is our convention. Note that this event has nothing to do with the `showBSF` parameter or whether the BSF chart is shown more broadly. As the relevant code in `wpt-app.js` indicates, it is guarding analytics event submission when the user is still interacting with the chart in order to avoid a known issue.

Reply:
Aha. Thanks for the explanation.